prompttools/mock/mock.py

```diff
@@ -125,3 +126,8 @@ def mock_mindsdb_completion_fn(**kwargs):
 
 def mock_lc_completion_fn(**kwargs):
     return "The first president of the United States was George Washington."
+
+
+def mock_stable_diffusion(**kwargs):
+    warnings.warn("Need access to Hugging Face or comparison images will be overwritten")
+    return None
```

Comment:
Question: Is it possible to return an image here?

If it is too much work we can either leave it as it is or skip having a mock function.